k0764427/quiz1.py

```diff
@@ -0,0 +1,28 @@
+############################
+#
+# Please complete this quiz
+#
+
+# declare an int variable named total with the value of 0
+
+# declare an array named numbers with all numbers 0 - 20
+
+# write a for loop that processes each number
+# if the number is even add it to total
+# if the number is odd and a multiple of 5 display the number preceded by the word Checkpoint: on a single line
+
+#display the value of total followed by a dash and the word total
+
+total = 0
+
+numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 , 13, 14, 15, 16, 17, 18, 19, 20]
+
+for i in numbers: 
+    if i % 2 == 0:
+        value = i
+        total = total + value
+    else:
+        if i % 5 == 0:
+            print "Checkpoint %d" % value
```

Comment:
Trace the values of your variables (like what we did in class today) through a couple of iteration s of your loop (at least enough to hit your else / i % 5 branch. This stack overflow is also relevant http://stackoverflow.com/questions/2829528/whats-the-scope-of-a-python-variable-declared-in-an-if-statement